### PR TITLE
Update supported cassandra versions

### PR DESCRIPTION
**Pull request by pezra**

### CODE REVIEW

File: .cassandra-versions [hunk 1]
```diff
@@ -1,108 +1,3 @@
...
-2.0.7
-2.0.8
-2.0.9
-2.0.10
-2.0.11
-2.0.12
-2.0.13
-2.0.14
-2.0.15
-2.0.16
```
Comment (outoftime) [inline .cassandra-versions]: @pezra this list should be auto-generated, and is intended to contain a list of *all* Cassandra versions, not just the supported ones—I believe there’s a line of code in the `Vagrantfile` that filters it down to the supported ones

Comment (outoftime) [inline .cassandra-versions]: [Here’s the task that generates it](https://github.com/cequel/cequel/blob/master/Rakefile#L122).

Comment (pezra) [inline .cassandra-versions]: oic... Thanks for the pointer.

Comment (outoftime) [inline .cassandra-versions]: No problem! I probably should’ve written the task to put a comment at the top of the file—not the first time this confusion has occurred

Comment (pezra) [inline .cassandra-versions]: The cassandra distro site is down for maintenance. I'll update the PR once it is back up.

File: README.md [hunk 2]
```diff
@@ -607,7 +608,7 @@ releases of Cequel.
 If you're new to Cassandra, check out [Learning Apache
 Cassandra](http://www.amazon.com/gp/product/1783989203/ref=s9_simh_co_p14_d4_i1?pf_rd_m=ATVPDKIKX0DER&pf_rd_s=left-1&pf_rd_r=1TX356WHGF06W32ZHD8S&pf_rd_t=3201&pf_rd_p=1953562742&pf_rd_i=typ01),
 a hands-on guide to Cassandra application development by example, written by
-the maintainer of Cequel.
+the creator of Cequel.
```
Comment (outoftime) [inline README.md]: 😜

**Pull request merged**